src/esp/assets/ResourceManager.cpp

```diff
@@ -124,6 +126,10 @@ void ResourceManager::buildImporters() {
 }  // buildImporters
 
 void ResourceManager::initDefaultPrimAttributes() {
+  if (!g_createMagnumRenderer) {
```

Comment:
Todo: issue a warning when users try to use primitives that didn't get loaded due to this code